Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling.yml

```diff
@@ -378,15 +393,183 @@ tasks:
+    scriptarguments:
+      SOCEmailAddress:
+        simple:
+      alert_id:
+        complex:
+          root: PaloAltoNetworksXDR.Incident.alerts
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: PaloAltoNetworksXDR.Incident.alerts.name
+                iscontext: true
+              right:
+                value:
+                  simple: Unusual allocation of multiple cloud compute resources
+              ignorecase: true
+          accessor: alert_id
+      incident_id:
+        complex:
+          root: inputs.incident_id
+      requireAnalystReview:
+        simple: "True"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": -560,
+          "y": 900
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 0aa062ec-14be-440b-8606-a05efe27dd8f
+    type: playbook
+    task:
+      id: 0aa062ec-14be-440b-8606-a05efe27dd8f
+      version: -1
+      name: GenericPolling
+      description: |-
+        Use this playbook as a sub-playbook to block execution of the master playbook until a remote action is complete.
+        This playbook implements polling by continuously running the command in Step \#2 until the operation completes.
+        The remote action should have the following structure:
+
+        1. Initiate the operation.
+        2. Poll to check if the operation completed.
+        3. (optional) Get the results of the operation.
+      playbookName: GenericPolling
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "12"
+    scriptarguments:
+      Ids:
+        simple: ${incident.xdrincidentid}
+      Interval:
+        simple: "1"
+      PollingCommandArgName:
+        simple: incident_id
+      PollingCommandName:
+        simple: 'xdr-get-incident-extra-data '
+      Timeout:
+        simple: "3"
+      dt:
+        simple: PaloAltoNetworksXDR.Incident.alerts.name(val.name!=='Unusual allocation of multiple cloud compute resources')
+    separatecontext: false
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": -400,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: 18ea3ba1-78ee-4ce3-844b-44853df600da
+    type: condition
+    task:
+      id: 18ea3ba1-78ee-4ce3-844b-44853df600da
+      version: -1
+      name: Is XCLOUD Cryptomining alert available?
```

Comment:
what do you mean by available?

Reply:
will change it to "Was XCLOUD Cryptomining alert found?"

Reply:
done